addons/web_editor/static/lib/odoo-editor/test/spec/color.test.js

```diff
@@ -0,0 +1,116 @@
+        });
+    });
+    it('should not merge line on background color change', async () => {
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><strong>[abcd</strong><br><strong>efghi]</strong></p>',
+            stepFunction: setColor('rgb(255, 0, 0)', 'backgroundColor'),
+            contentAfter: '<p><strong><font style="background-color: rgb(255, 0, 0);">[abcd</font></strong><br>' +
+                          '<strong><font style="background-color: rgb(255, 0, 0);">efghi]</font></strong></p>',
+        });
+    });
+    it('should not merge line on color change', async () => {
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><strong>[abcd</strong><br><strong>efghi]</strong></p>',
+            stepFunction: setColor('rgb(255, 0, 0)', 'color'),
+            contentAfter: '<p><strong><font style="color: rgb(255, 0, 0);">[abcd</font></strong><br>' +
+                          '<strong><font style="color: rgb(255, 0, 0);">efghi]</font></strong></p>',
+        });
+    });
+    it('should remove font tag after removing font color', async () => {
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><font style="color: rgb(255, 0, 0);">[abcabc]</font></p>',
+            stepFunction: setColor('', 'color'),
+            contentAfter: '<p>[abcabc]</p>',
+        });
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><font class="text-400">[abcabc]</font></p>',
+            stepFunction: setColor('', 'color'),
+            contentAfter: '<p>[abcabc]</p>',
+        });
+    });
+    it('should remove font tag after removing background color applied as style', async () => {
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><font style="background-color: rgb(255, 0, 0);">[abcabc]</font></p>',
+            stepFunction: setColor('', 'backgroundColor'),
+            contentAfter: '<p>[abcabc]</p>',
+        });
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><font class="bg-200">[abcabc]</font></p>',
+            stepFunction: setColor('', 'backgroundColor'),
+            contentAfter: '<p>[abcabc]</p>',
+        });
+    });
+    it('should remove font tag if font-color and background-color both are removed one by one', async () => {
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><font style="color: rgb(255, 0, 0);" class="bg-200">[abcabc]</font></p>',
+            stepFunction: setColor('','backgroundColor'),
+            stepFunction: setColor('','color'),
+            contentAfter: '<p>[abcabc]</p>',
+        });
+        await testEditor(BasicEditor, {
+            contentBefore: '<p><font style="background-color: rgb(255, 0, 0);" class="text-900">[abcabc]</font></p>',
+            stepFunction: setColor('','color'),
+            stepFunction: setColor('','backgroundColor'),
+            contentAfter: '<p>[abcabc]</p>',
+        });
+    });
+});
```

Comment:
This is not the correct file. The editor has been moved in this version and is not under lib anymore. Please make the changes in the right file.

Reply:
@dmo-odoo Updated the PR. Please have a look.